src/fidesops/models/privacy_request.py

```diff
@@ -378,7 +377,9 @@ def start_processing(self, db: Session) -> None:
         """Dispatches this PrivacyRequest throughout the Fidesops System"""
         if self.started_processing_at is None:
             self.started_processing_at = datetime.utcnow()
-            self.save(db=db)
+        if self.status == PrivacyRequestStatus.pending:
+            self.status = PrivacyRequestStatus.in_processing
+        self.save(db=db)
```

Comment:
If a privacy request does not need approval, it seems like this should go to in_processing when it starts processing instead of staying on 'pending'